app/main.py

```diff
@@ -1,3 +1,26 @@
-def shop_trip():
-    # write your code here
-    pass
+from json import load
+from app.calculate import can_afford_trip
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = load(file)
+    fuel_price = data.get("FUEL_PRICE", 0)
+    customers = []
+    for customer in data.get("customers", 0):
```

Comment:
if "customers" will not exist you can get error: Integer is not iterable. Choose better default argument, the same in shops